language/vm/vm_runtime/src/txn_executor.rs

```diff
@@ -252,15 +252,27 @@ where
                     .ok_or(VMInvariantViolation::LinkerError)?;
 
                     if callee_function_ref.is_native() {
-                        let module_name: &str = callee_function_ref.module().name();
+                        let module = callee_function_ref.module();
```

Comment:
We had `ModuleId` that acts like a unique identifier for a module. We probably should use that instead of passing in the name and address as this identifier might subject to change after we have versioning

Reply:
Done